user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Indexing_database/OpenSearch_Database/Configuring_OpenSearch_Backups.md

```diff
@@ -46,126 +54,127 @@ Below we are talking about NFS server and NFS clients. The NFS server is respons
 
-  1. Another test is creating a file:
+1. Create a test file in the mounted directory to verify accessibility:
 
-      ```bash
-      sudo touch /var/nfs/opensearch/test.txt
-      ```
+   ```bash
+   sudo touch /var/nfs/opensearch/test.txt
+   ```
 
-      This should create a file named test.txt which can be seen from any other NFS client.
+   This will create a file named "test.txt" that can be accessed from any other NFS client.
 
-      > [!TIP]
-      > If required, contact your IT department to follow the NFS server setup procedure, as this step may otherwise be challenging. Further input on networking infrastructure may also be required.
+### Creating a snapshot repository
 
-      > [!TIP]
-      > It is also possible to set up an NFS share on Windows and share it with the destination Linux system. See [Windows Server 2016 as an NFS server for Linux clients](https://blog.bobbyallen.me/2018/01/18/windows-server-2016-as-an-nfs-server-for-linux-clients/).
+#### Creating a repository using OpenSearch Dashboards
 
-      > [!NOTE]
-      >
-      > - As this setup might be more challenging, we recommend that you set it up in advance.
-      > - Be wary of read, write, and execute rights, firewall configurations, and the state of the NFS server service during the setup. For the machines that will act as NFS servers, make sure the firewall allows NFS traffic. You can use the following command for this: *sudo ufw allow nfs*.
-      > - Make sure the OpenSearch user has enough rights to the folder to read, write, and execute its contents.
+> [!IMPORTANT]
+> To be able to restore a snapshot later, you must go through these steps first. Without a functional repository, you will run into issues that will delay the snapshot restore.
+
+1. Access the *opensearch.yml* file on all nodes of the OpenSearch cluster. Open the file and set *path.repo* as your previously created shared folder.
 
-### Create a repository via OpenSearch Dashboards GUI
+1. Open OpenSearch Dashboards, and navigate to the left-hand pane. Under *Snapshot Management*, select *Repositories* to view an overview of all repositories.
 
-1. On the OpenSearch cluster, on all nodes, navigate to the *opensearch.yml* file. Open the file and set *path.repo* as your previously created shared folder.
+1. Select *Create repository* in the top-right corner.
 
-1. In OpenSearch Dashboards go to Snapshot Management - Repositories:
+   ![OpenSearch Dashboards - Repository Overview ](~/user-guide/images/OpenSearchDashboards_RepositoryOverview.png)
 
-   ![OpenSearch Dashboards - Repositories ](~/user-guide/images/OpenSearchDashboards_Repositories.png)
+1. Specify the following details:
 
-1. This will open a **Repositories Overview**:
+   - **Repository name**: The name of your repository
 
-   ![OpenSearch Dashboards - Repository Overview ](~/user-guide/images/OpenSearchDashboards_RepositoryOverview.png)
+   - **Location**: The location where your repository is saved
```

Comment:
Could you provide me with the correct settings used in the UI?

Reply:
normally these are the settings, you mean provide a screenshot?

Reply:
On lines 242-245, I was able to type out the correct setting names as I had a screenshot to copy them down from: 

- **Snapshot name**: The name of the snapshot
- **Select or input source indexes or index patterns**: Enter an asterisk ("*")
- **Select a repository for snapshots**: The repository you created in Creating a snapshot repository

I don't necessarily need a screenshot here, but wanted some assurance that these are the exact setting names. Is it "Location", as I've written here, or "Select a location for snapshots", for instance?